coderd/database/queries/provisionerdaemons.sql

```diff
@@ -69,6 +72,10 @@ LEFT JOIN
 			LIMIT 1
 		)
 	)
+JOIN
```

Comment:
```suggestion
LEFT JOIN
```
Let's also COALESCE the fields so we don't need to handle nulls.

Reply:
How should I use COALESCE in this case?